### PR TITLE
Added documentation for integration with AZ Devops, CircleCI and Bitrise

### PR DESCRIPTION
**Pull request by ahmedajaj**

### CODE REVIEW

File: docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+- `RP_INSTANCE_URL`: The URL of your ReportPortal instance.
+- `RP_PROJECT`: The ReportPortal project name.
+- `REQUEST_TIMEOUT_SECONDS`: Timeout for ReportPortal API requests.
+- `SCRIPT_TIMEOUT_SECONDS`: Timeout for the entire script execution.
+
+In the **Variables** section, add the following sensitive variable:
+- `RP_API_KEY`: Your API key from ReportPortal.
+
+### Step 1: Run application build and tests
```
Comment (HardNorth) [inline docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md:38]: Please add a paragraph where to put pipeline configuration file and how to point Azure DevOps on it.
As here: https://reportportal.io/docs/quality-gates/IntegrationWithCICD/IntegrationWithGitHubActions#using-github-actions-workflow-file

File: docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Use the following scripts to run tests with npm. The test execution logs are captured in `./console.log`. Then the ReportPortal Launch ID is extracted and saved in `RP_LAUNCH_ID`. The script you choose depends on whether you are using Bash or Powershell. 
+
+#### Bash Script
+
+```yaml
+- script: |
```
Comment (HardNorth) [inline docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md:45]: The step should not fail if we talk about Quality Gates configuration. You put try-catch in powershell script but left bash script without such option.

File: docs/quality-gates/IntegrationWithCICD/IntegrationWithCircleCI.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Define these environment variables in your CircleCI project settings:
+
+- `RP_INSTANCE_URL`: The URL of your ReportPortal instance.
+- `RP_PROJECT`: The name of your project as it is in ReportPortal.
+- `RP_API_KEY`: Your API key from ReportPortal.
+- `REQUEST_TIMEOUT_SECONDS`: The timeout duration for ReportPortal API requests.
+- `SCRIPT_TIMEOUT_SECONDS`: The timeout duration for the entire script execution.
+
+### CircleCI Configuration
```
Comment (HardNorth) [inline docs/quality-gates/IntegrationWithCICD/IntegrationWithCircleCI.md:36]: Please split it into steps with explanations as in other articles.

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md (hunk 1 above)
Comment (ahmedajaj) [reply docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md:53]: Fixed please check

File: docs/quality-gates/IntegrationWithCICD/IntegrationWithCircleCI.md (hunk 3 above)
Comment (ahmedajaj) [reply docs/quality-gates/IntegrationWithCICD/IntegrationWithCircleCI.md]: Fixed please check

File: docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md (hunk 2 above)
Comment (ahmedajaj) [reply docs/quality-gates/IntegrationWithCICD/IntegrationWithAzureDevOps.md:60]: Fixed please check

**Review by HardNorth — APPROVED** [review]

**Comment by HardNorth** [comment]
@pressayuliya Please review.

**Review by pressayuliya — APPROVED** [review]

**Pull request merged**